src/App/Utilities/constants.js

```diff
@@ -0,0 +1,288 @@
+import React from "react";
+import { spacingClasses, borderClasses, breakpointClasses, hidingElementClasses, flexClasses, imageClasses, sizingClasses, textClasses, visibilityClasses } from "./classes";
+import CodeTagComponent from "@components/CodeTags";
+
+const UtilityPresenter = ({ util, css, example, breakpointVersion }) => (
+    <div className="d-flex justify-content-between mb-2">
+        <CodeTagComponent code={util} type="primary"/>
+        <CodeTagComponent code={css} type="secondary"/>
+        {breakpointVersion ? <CodeTagComponent type="tertiary" code={example}/> : <>{example && <span>{example}</span>}</>}
+    </div>
+);
+
+const DisplayPresenter = ({ util, css, textVersion }) => (
+    <div className="d-flex justify-content-between mb-2">
+        {textVersion ? <span>{util}</span> : <CodeTagComponent code={util} type="primary"/>}
+        <CodeTagComponent code={css} type="secondary"/>
+    </div>
+);
+
+const FlexPresenter = ({ code }) => (
+    <div className="d-flex  mb-2">
+        <CodeTagComponent code={code} type="primary"/>
+    </div>
+);
+
+export const borders = [
+    {
+        id: "borders",
+        title: "Border",
+        subtitle: "Gives borders to an element.",
+        content: <>
+            {borderClasses.map((rule, i) => (
+                <UtilityPresenter key={rule + i} util={rule.name} css={rule.css}/>
+            ))}
+        </>
+    }];
+
+export const breakpoints = [
+    {
+        id: "breakpoints",
+        title: "Breakpoints",
+        subtitle: "Break it up",
+        content: <>
+            <div className="d-flex justify-content-between">
+                <p className="font-weight-bold">Infix</p>
+                <p className="font-weight-bold">Min-width</p>
+                <p className="font-weight-bold">Example use</p>
+            </div>
+            {breakpointClasses.map((rule, i) => (
+                <UtilityPresenter breakpointVersion key={rule + i} util={rule.name} css={rule.css} example={rule.px}/>
+            ))}
```

Comment:
should this be displayed as a CSS grid instead of flex, so there is a dynamic column alignment
(can be done in an iteration 💅)
![image](https://user-images.githubusercontent.com/15030804/211760150-e2581774-8ade-44a2-96f0-6aa7704ef013.png)